msticpy/nbtools/timeline.py

```diff
@@ -868,10 +865,16 @@ def _create_range_tool(
 # pylint: enable=too-many-arguments
 
 
-def _add_ref_line(plot, ref_time, ref_text="Ref time", series_count=1):
+def _add_ref_line(plot, ref_time, ref_text="Ref time", series_count=100):
```

Comment:
Not sure why the default of 100 is needed. I think I only put a default there because the ref_text param has a default and you can't have a subsequent param without a default. I think in each case we call this, we specify the value.

Reply:
Had some feedback from user saying that the line 'never showed'. Turned out it was showing but because it was len of group_by data set rather than related to the y_axis value size it was creating a very small line instead. This change was part of testing what was happening. Have included the actual fix now which takes the max y_axis value and uses that as the series_count size to pass to _add_ref_line.